test/MUXControlsTestApp/Utilities/EntityPropertiesControl/EntityPropertiesControl.xaml.cs

```diff
@@ -147,6 +149,9 @@ protected virtual void GenerateUI(PropertyInfo propertyInfo, out FrameworkElemen
 
         private void GeneratingUI()
         {
+#if ARM64 // CS7069: Reference to type 'BindingFlags' claims it is defined in 'System.Reflection', but it could not be found                    
```

Comment:
Weren't we told this error was because we were targeting a version of .NET that doesn't actually support arm64?

Reply:
updated .net version